EIPS/eip-7515.md

```diff
@@ -0,0 +1,123 @@
+---
+eip: 7515
+title: NFT Alternative Text in Metadata
+description: Populate alternative text field in NFT Metadata with image description for screen reader.
+author: Mona Rassouli (@DecoratedWings)
+discussions-to: https://ethereum-magicians.org/t/proposal-for-informational-eip-simple-accessibility-recommendation/14639
+status: Draft
+type: Informational
```

Comment:
```suggestion
type: ERC
```
Given that this is application layer, it should be an ERC.

Reply:
Thank you for the correction!